File: benchmark/template/programs/go/adversarial/ADV-25_R.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	fmt.Println("ADV-START")

	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	n, _ := strconv.ParseInt(sc.Text(), 10, 64)

	r := n * 2

	fmt.Printf("OBS=R:%d\n", r)
	fmt.Println("ADV-END")
}

File: benchmark/template/programs/cpp/adversarial/ADV-25_R.cpp
#include <cstdint>
#include <iostream>
#include <string>

int main() {
    std::cout << "ADV-START" << std::endl;
    std::string t;
    std::getline(std::cin, t);
    std::int64_t r = std::stoll(t) * 2;
    std::cout << "OBS=R:" << r << std::endl;
    std::cout << "ADV-END" << std::endl;
    return 0;
}
